external-crates/move/crates/move-compiler/tests/move_check/translated_ir_tests/move/commands/abort_negative_stack_size.move

```diff
@@ -1,7 +0,0 @@
-// check: NEGATIVE_STACK_SIZE_WITHIN_BLOCK
```

Comment:
Removed this test as it was wrong (and definitely not testing what it says).

Reply:
Its just a translated test... but sure